libkodimote/libkodimote.pro

```diff
@@ -3,7 +3,7 @@ contains(QT_VERSION, ^5\\..\\..*) {
     DEFINES += QT5_BUILD
     QT += quick qml
 } else {
-    QT += declarative
+    QT += declarative network
```

Comment:
Wow. I've remembered I put the "network" one above the condition. But checking I now see I did: `Qt += concurent network`, so `Qt` (lowercase `t`) instead of `QT` (capital `t`). Could you check if it works ok without this change, but chaning the `Qt` to `QT` above? This might also make the change in `imagecache.cpp` (`#include <QtCore>` for non Qt 5 builds) obsolete, as I guess the module isn't included properly now because of this typo.